### ISSUE TITLE
Behavioral Cloning - 16 Visualizing Loss import

### ISSUE THREAD

**Issue by josemacenteno** [udacity/sdc-issue-reports#591]
The code has an error onthe first line. 

It is:
import keras.models import Model

I think the correct code should be:
from keras.models import Model

**Comment by swwelch** [comment]
Thanks! Fixed now.
